### ISSUE TITLE
Broken on Textual v0.11+

### ISSUE THREAD

**Issue by ofek** [mitosch/textual-select#1]
- https://github.com/Textualize/textual/blob/cb84d9111c48c8cc8cccab4bfd867d6eb88b68ae/CHANGELOG.md#removed-1
- https://github.com/Textualize/textual/pull/1738

https://github.com/mitosch/textual-select/blob/main/examples/dropdowns.py

```
╭───────────────────────────────────────────────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────────────────────────────────────────────╮
│ C:\Users\ofek\AppData\Local\Programs\Python\Python38\lib\site-packages\textual_select\_select.py:327 in watch_value                                                                                 │
│                                                                                                                                                                                                     │
│   324 │   │   │   self.text = ""                                                               ╭────────────────── locals ──────────────────╮                                                       │
│   325 │   │   │   self.select_list.list_view.index = 0                                         │  self = Select(pseudo_classes={'enabled'}) │                                                       │
│   326 │   │   │   self.refresh(layout=True)                                                    │ value = 0                                  │                                                       │
│ ❱ 327 │   │   await self.emit(self.Changed(self, value))                                       ╰────────────────────────────────────────────╯                                                       │
│   328 │                                                                                                                                                                                             │
│   329 │   class Changed(Message, bubble=True):                                                                                                                                                      │
│   330 │   │   """Value was changed."""                                                                                                                                                              │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
AttributeError: 'Select' object has no attribute 'emit'
```

**Comment by mitosch** [comment]
Thank you for reporting. I will have a look at it and publish an update as soon as possible.

**Comment by mitosch** [comment]
Fixed for textual-select 0.3.0 which requires textual >= 0.11.0. For older textual versions, textual-select 0.2.2 can be used.
